website/docs/d/public_ip.html.markdown

```diff
@@ -15,134 +15,16 @@ Use this data source to access the properties of an existing Azure Public IP Add
 
 ```hcl
 data "azurerm_public_ip" "datasourceip" {
-    name = "testPublicIp"
-    resource_group_name = "acctestRG"
+    name = "name_of_public_ip"
+    resource_group_name = "name_of_resource_group"
 }
 
-resource "azurerm_virtual_network" "helloterraformnetwork" {
-    name = "acctvn"
-    address_space = ["10.0.0.0/16"]
-    location = "West US 2"
-    resource_group_name = "acctestRG"
+output "public_ip_address_id" {
+    domain_name_label = "${data.azurerm_public_ip.datasourceip.id}"
 }
 
-resource "azurerm_subnet" "helloterraformsubnet" {
-    name = "acctsub"
-    resource_group_name = "acctestRG"
-    virtual_network_name = "${azurerm_virtual_network.helloterraformnetwork.name}"
-    address_prefix = "10.0.2.0/24"
-}
-
-resource "azurerm_network_interface" "helloterraformnic" {
-    name = "tfni"
-    location = "West US 2"
-    resource_group_name = "acctestRG"
-
-    ip_configuration {
-        name = "testconfiguration1"
-        subnet_id = "${azurerm_subnet.helloterraformsubnet.id}"
-        private_ip_address_allocation = "static"
-        private_ip_address = "10.0.2.5"
-        public_ip_address_id = "${data.azurerm_public_ip.datasourceip.id}"
-    }
-}
-```
-
-## Example Usage (Retrieve the Dynamic Public IP of a new VM)
```

Comment:
this is a topic which comes up fairly frequently - as such could we add this back in?

Reply:
Agree, we are planning to make a full article on docs on this topic to guide customers on how to use it.

Reply:
could we restore this sample for the moment - and then when the guide is published we should be able to replace this sample with a link to the guide?

Reply:
Fully agree, I am working on the docs this week and will update this PR when done. Will notify you.

Reply:
@dcaro any chance we could restore this second example? this otherwise looks good to me :)